Makefile

```diff
@@ -1,76 +1,36 @@
-.PHONY: all clean install build test libinstall reinstall uninstall distclean reindent
-all: build
+include config.mk
 
-NAME=xenopsd
-J=4
+OPAM_PREFIX=$(DESTDIR)$(shell opam config var prefix)
+OPAM_LIBDIR=$(DESTDIR)$(shell opam config var lib)
 
-ENABLE_TESTS=--enable-tests
-COVERAGE=coverage
+.PHONY: build clean release test reindent install uninstall
 
-clean:
-	@rm -f setup.data setup.log setup.bin setup.ml _oasis lib/version.ml
-	@rm -rf _build
-	@rm -f xenopsd-xc xenopsd-xenlight xenopsd-simulator xenopsd-libvirt
-	@rm -f xenopsd-xc.1 xenopsd-xenlight.1 xenopsd-simulator.1 xenopsd-libvirt.1
-	@rm -f *.native
-	@rm -f /tmp/bisect-xenops*.out || true
-	@rm -rf $(COVERAGE)
-
--include config.mk
-
-config.mk:
-	echo Please re-run configure
-	exit 1
+build:
+	jbuilder build @install
```

Comment:
We should probably reconsider this. I'd be cleaner if in the spec files we just invoke `make`, and during development we manually do `make build`. But I guess this can be changed and made uniform in a later stage

Reply:
How about implementing Gabor's suggestion and marking the `release` target as the default?

Reply:
I had a quick look at buildig with `--dev` on and the number of warnings is pretty high. I'd rather fix them (because disabling them individually defeats the point of dev), however I think it should be done separately from this PR and after the `qemu-upstream` branch has been rebased on this, because I would like to minimise conflicts as much as possible.

Reply:
I agree with all the points you make